core/state/database.go

```diff
@@ -42,10 +42,9 @@ const (
 
 // Database wraps access to tries and contract code.
 type Database interface {
-	// Arbitrum: CompiledWasmContractCode retrieves a particular contract's user wasm code.
-	CompiledWasmContractCode(version uint16, codeHash common.Hash) ([]byte, error)
-	// Arbitrum: SetCompiledWasmContractCode sets a user wasm code.
-	SetCompiledWasmContractCode(version uint16, codeHash common.Hash, code []byte) error
+	// Arbitrum: Manage activated Stylus contracts
+	ActivatedAsm(moduleHash common.Hash) (asm []byte, err error)
+	ActivatedModule(moduleHash common.Hash) (module []byte, err error)
```

Comment:
nit: I think the comment should be "Read activated Stylus contracts", as these functions on read and not write

Reply:
Nice catch -- this used to be accurate but no longer is :)